localization/english/bookmark/bookmark_605_l_english.yml

```diff
@@ -1,48 +1,34 @@
 
  #Kael'thas Sunstrider
  kaelthas_character:0 "Prince Kael'Thas"
- kaelthas_character_desc:0 "Crown-prince of Quel'Thalas and archmage of Dalaran, Kael'Thas Sunstrider lost both of his homes to Arthas and the Scourge during the Third War. With his kingdom destroyed and his father dead, Kael renounced the title of king but still gathered other elven survivors, renaming them Blood Elves in honor of their fallen brethren. Soon, the prince discovered that his people was not only threatened of extinction by the Scourge, but also by its own addiction to magic causing an insatiable thirst since the Sunwell, source of Quel'Thalas's power, was lost. Looking for allies, the prince first pledged himself to the Alliance, but desperation, magical thirst and Alliance ruler Garithos's hostility against the elves forced Kael'Thas to ally with the Illidari and be accused of treason by their former human allies. The rogue demon Illidan Stormrage offered Kael and his people new sources of magic in exchange for their loyalty, an offer that could not be ignored. Now in the service of Illidan, Kael'Thas is engaged along with his new master in a quest to destroy the Lich-King, an opportunity to avenge the fall of Quel'Thalas.
- 
- Engaged in a war with the Lich-King, will Kael'Thas succumb to the darkness of the Scourge ? Will he be led to damnation by his pact with Illidan and the Legion ? Or will he bring glory to the blood elves again ? "
+ kaelthas_character_desc:0 "Crown-prince of Quel'Thalas and archmage of Dalaran, Kael'Thas Sunstrider lost both of his homes to Arthas and the Scourge during the Third War. With his kingdom destroyed and his father dead, Kael renounced the title of king but still gathered other elven survivors, renaming them Blood Elves in honor of their fallen brethren. Soon, the prince discovered that his people was not only threatened of extinction by the Scourge, but also by its own addiction to magic causing an insatiable thirst since the Sunwell, source of Quel'Thalas's power, was lost. Looking for allies, the prince first pledged himself to the Alliance, but desperation, magical thirst and Alliance ruler Garithos's hostility against the elves forced Kael'Thas to ally with the Illidari and be accused of treason by their former human allies. The rogue demon Illidan Stormrage offered Kael and his people new sources of magic in exchange for their loyalty, an offer that could not be ignored. Now in the service of Illidan, Kael'Thas is engaged along with his new master in a quest to destroy the Lich King, an opportunity to avenge the fall of Quel'Thalas. Engaged in a war with the Lich King, will Kael'Thas succumb to the darkness of the Scourge ? Will he be led to damnation by his pact with Illidan and the Legion ? Or will he bring glory to the blood elves again ? "
```

Comment:
```suggestion
 kaelthas_character_desc:0 "Crown-prince of Quel'Thalas and archmage of Dalaran, Kael'Thas Sunstrider lost both of his homes to Arthas and the Scourge during the Third War. With his kingdom destroyed and his father dead, Kael renounced the title of king but still gathered other elven survivors, renaming them Blood Elves in honor of their fallen brethren. Soon, the prince discovered that his people was not only threatened of extinction by the Scourge, but also by its own addiction to magic causing an insatiable thirst since the Sunwell, the source of Quel'Thalas's power, was lost. Looking for allies, the prince first pledged himself to the Alliance, but desperation, magical thirst, and Alliance ruler Garithos's hostility against the elves forced Kael'Thas to ally with the Illidari and be accused of treason by their former human allies. The rogue demon Illidan Stormrage offered Kael and his people new sources of magic in exchange for their loyalty, an offer that could not be ignored. Now in the service of Illidan, Kael'Thas is engaged along with his new master in a quest to destroy the Lich King, an opportunity to avenge the fall of Quel'Thalas. Engaged in a war with the Lich King, will Kael'Thas succumb to the darkness of the Scourge? Will he be led to damnation by his pact with Illidan and the Legion? Or will he bring glory to the blood elves again? "
```